detection-rules/attachment_eml_file_with_html_attachment.yml

```
@@ -0,0 +1,31 @@
+name: "Attachment: EML file with HTML attachment (first-time sender)"
+description: |
+  Potential HTML smuggling via HTML file inside embedded rfc822 (eml)
+type: "query"
+severity: "medium"
```

```suggestion
severity: "low"
```

wdyt about `low` here, since we're not looking for any specific smuggling signals?